eruchristo/.htaccess

```diff
@@ -0,0 +1,4 @@
+Options +FollowSymLinks
+RewriteEngine on
+RewriteRule ^eruchristo/?(.*) https://www.christopherkrueger.com/docs/eruchristo.html
```

Comment:
you are minting https://w3id.org/eurochristo/eurochristo. Is this intended or a typo?